mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1376,6 +1378,23 @@
     <!-- Text shown in the notification that pops up to remind the user that a private browsing session is active. -->
     <string name="notification_pbm_delete_text_2">Close private tabs</string>
 
+    <!-- Microsuverys -->
+    <!-- Text shown in prompt for printing microsurvey. -->
+    <string name="microsurvey_prompt_printing_title" tools:ignore="UnusedResources">Help make printing in Firefox better. It only takes a sec</string>
```

Comment:
@Amejia481 Could you give a brief comment as to what "sec" means, for localizers please?

Reply:
Sorry for the delay I filed https://phabricator.services.mozilla.com/D212175 to add the suggestions.